sr_unix/gtm_getpwuid.c

```diff
@@ -51,18 +51,14 @@ struct passwd	*gtm_getpwuid(uid_t uid)
 		if (-1 == buff_size)
 			buff_size = 2048;
 		buff = malloc(buff_size);
-		if (NULL == buff)
-		{
-			return NULL;
-		}
 		getpwuid_r(uid, &temp_passwd, buff, buff_size, &retval);
```

Comment:
It was  an interim state. It caused most of the coverity project issues that you found and sent me mail about because it trashed the userid that got put in the journal records so many things that compared extracts failed. Since it's largely irrelevant now, I'll just remove it to reduce confusion.